src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { TodoType } from '../../types/TodoType';
+import { TodoInfo } from '../TodoInfo';
+
+type TodoListProps = {
+  todos: TodoType[];
+};
+
+export const TodoList: React.FC<TodoListProps> = ({ todos }) => {
+  return (
```

Comment:
as you don't have any logic before return - we should make right-away return statement in our arrow funciton
```suggestion
export const TodoList: React.FC<TodoListProps> = ({ todos }) => (
```